src/client/firefox/commands.js

```diff
@@ -118,40 +122,23 @@ function setBreakpoint(
       condition,
       noSliding
     })
-    .then((res: BreakpointResponse) => onNewBreakpoint(location, res));
+    .then(([{ actualLocation }, bpClient]) => {
+      actualLocation = createBreakpointLocation(location, actualLocation);
```

Comment:
This isn't wrong in terms of logic, but I find it difficult to follow if args are reassigned in code. Generally, I would avoid reassignment due to how it impacts code reading. In this case, it's ok because we don't have `if` statements, but as a preventative measure, I think it would be better to use a new variable here.  In the lower case where we are reassigning `bpClients[i]`, it's unavoidable, so that can stay as it is.